oak_runtime/src/lib.rs

```diff
@@ -1164,6 +1165,26 @@ impl Runtime {
         config: &NodeConfiguration,
         label: &Label,
         initial_handle: oak_abi::Handle,
+    ) -> Result<(), OakStatus> {
+        // This only creates a Node instance, but does not start it.
+        let instance = self.node_factory.create_node(config).map_err(|err| {
+            warn!("could not create node: {:?}", err);
+            OakStatus::ErrInvalidArgs
+        })?;
+
+        // Register the instance within the `Runtime`.
+        self.register_node_instance(node_id, instance, config, label, initial_handle)
+    }
+
+    /// Registers the given [`Node`] instance within the [`Runtime`]. The registration fails if the
+    /// labels violate the IFC rules.
+    fn register_node_instance(
```

Comment:
Maybe the other method should be called `node_create_and_register` and this `node_register`, or some other combination of names that are consistent with each other.